### ISSUE TITLE
Adding location as favorite, shows old name.

### ISSUE THREAD

**Issue by Randshot** [jaszhix/NoMansConnect#90]
I have a location, that I renamed **thrice**. Now, when I add it as favorite, it shows the *second* name I gave it.
After restarting NMC, it is fixed again.
Don't know what is causing it.

I have tried the following:
- Syncing locations
- Resetting remote cache
- Deleting NMC folder in `AppData\Roaming`
- Reinstalling NMC

**Comment by jaszhix** [comment]
Can you please attach NMC.log and settings.json? Thanks.

**Comment by jaszhix** [comment]
I'm not able to reproduce this with the way you've shown it - I checked on the server and it looks like the name is "Home Planet" in the json data, but the proper name field for it contains "Novton: Mudium". I'm not sure how that could happen. Its also not clear to me from your issue which name you expect to be correct. I explicitly mismatched the name key value (which shouldn't exist) in a location's JSON data, it still displayed the server's version - which should be the version that is updated by the name change. Not sure what circumstance could have caused this, but that's why I ask for logs and settings.

**Comment by jaszhix** [comment]
If you can, please try this [pre-release version](https://github.com/jaszhix/NoMansConnect/releases/tag/v1.3.3-rc).

**Comment by Randshot** [comment]
I will try it with the pre-release version again, and also provide a log and settings file, if it is still happening.

**Comment by Randshot** [comment]
Ok, I have tried adding the location to my favorites like I did above, and the name still changes on adding it and also after restarting the application.

The original name I gave the location was `Novton: Mudium`, then I changed it to `Home Planet` and then back to `Novton: Mudium`. I guess something got out of sync in between?

Below I have attached the log and settings file.
[files.zip](https://github.com/jaszhix/NoMansConnect/files/2267041/files.zip)

**Comment by jaszhix** [comment]
Yes, it looks like the stored object version is incorrect, and not syncing correctly - I've been pushing a lot of clients lately, it could be a migration related bug - but I'm not able to reproduce this with other locations I tested. I can fix it on the server, and you can let me know if this happens with another location.
